src/mon/PGMap.h

```diff
@@ -439,11 +439,11 @@ class PGMap : public PGMapDigest {
   int64_t get_rule_avail(const OSDMap& osdmap, int ruleno) const;
   void get_rules_avail(const OSDMap& osdmap,
 		       std::map<int,int64_t> *avail_map) const;
-  void dump(ceph::Formatter *f) const; 
+  void dump(ceph::Formatter *f, bool with_net = true) const;
```

Comment:
since we have other commands for inspecting network stats, I'm not sure it makes sense to include in e.g. 'ceph pg dump' - it's a lot of extra info

Reply:
@jdurgin As we discussed you'd only get the network stats if requesting the JSON information.